studio-server/src/main/java/com/coremedia/labs/plugins/adapters/dropbox/server/DropboxContentHubAdapter.java

```diff
@@ -1,13 +1,6 @@
 package com.coremedia.labs.plugins.adapters.dropbox.server;
 
-import com.coremedia.contenthub.api.ContentHubAdapter;
-import com.coremedia.contenthub.api.ContentHubContext;
-import com.coremedia.contenthub.api.ContentHubObject;
-import com.coremedia.contenthub.api.ContentHubObjectId;
-import com.coremedia.contenthub.api.ContentHubTransformer;
-import com.coremedia.contenthub.api.Folder;
-import com.coremedia.contenthub.api.GetChildrenResult;
-import com.coremedia.contenthub.api.Item;
+import com.coremedia.contenthub.api.*;
```

Comment:
somebody within CM always told to not do * imports. Does it make sense to change it?

Reply:
Fixed. Stupid IDEA auto-did that because I didn't have the CoreMedia Code Style activated.